.ci/openshift-ci/buildall_install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build kata-containers and install into a given destination directory
+#
+
+set -e
+
+if [ -z "$1" ];then
+	echo "Usage: $0 path/to/install/dir"
```

Comment:
Nit: `>&2`. But maybe better to add this check after sourcing the `lib.sh` below. Then you can use our more idiomatic:

```sh
[ -z "$1" ] && die "$0 path/to/install/dir"
```

Reply:
Done